doc/sphinx/source/quickstart/installation.rst

```diff
@@ -661,6 +516,135 @@ that is particularly suited for new users that have an access to pre-installed v
+
+For example, the following command would run a recipe
+
+.. code-block:: bash
+
+   docker run -e HOME -v "$HOME":"$HOME" -v /data:/data esmvalgroup/esmvaltool:stable run examples/recipe_python.yml
+
+with the environmental variable ``$HOME`` available inside the container and
+the data in the directories ``$HOME`` and ``/data``, so these can be used to
+find the configuration file, recipe, and data.
+
+It might be useful to define a `bash alias
+<https://opensource.com/article/19/7/bash-aliases>`_
+or script to abbreviate the above command, for example
+
+.. code-block:: bash
+
+   alias esmvaltool="docker run -e HOME -v $HOME:$HOME -v /data:/data esmvalgroup/esmvaltool:stable"
+
+would allow using the ``esmvaltool`` command without even noticing that the
+tool is running inside a Docker container.
+
+.. _install_with_singularity:
+
+Singularity installation
+========================
+
+Docker is usually forbidden in clusters due to security reasons. However,
+there is a more secure alternative to run containers that is usually available
+on them: `Singularity <https://sylabs.io/guides/3.0/user-guide/quick_start.html>`_.
+
+Singularity can use docker containers directly from DockerHub with the
+following command
+
+.. code-block:: bash
+
+   singularity run docker://esmvalgroup/esmvaltool:stable run examples/recipe_python.yml
+
+Note that the container does not see the data available in the host by default.
+You can make host data available with ``-B /path:/path/in/container``.
+
+It might be useful to define a `bash alias
+<https://opensource.com/article/19/7/bash-aliases>`_
+or script to abbreviate the above command, for example
+
+.. code-block:: bash
+
+   alias esmvaltool="singularity run -B $HOME:$HOME -B /data:/data docker://esmvalgroup/esmvaltool:stable"
+
+would allow using the ``esmvaltool`` command without even noticing that the
+tool is running inside a Singularity container.
+
+Some clusters may not allow to connect to external services, in those cases
+you can first create a singularity image locally:
+
+.. code-block:: bash
+
+   singularity build esmvaltool.sif docker://esmvalgroup/esmvaltool:stable
+
+and then upload the image file ``esmvaltool.sif`` to the cluster.
+To run the container using the image file ``esmvaltool.sif`` use:
+
+.. code-block:: bash
+
+   singularity run esmvaltool.sif run examples/recipe_python.yml
+
+.. _install_with_pip:
+
+Pip installation
+================
+
+It is also possible to install ESMValTool from
+`PyPI <https://pypi.org/project/ESMValTool/>`_.
+However, this requires first installing dependencies that are not available
+on PyPI in some other way.
+The list of required dependencies can be found in
+:download:`environment.yml <../../../../environment.yml>`.
+
+.. warning::
+
+    It is recommended to use the installation with mamba instead, as it may not
+    be easy to install the correct versions of all dependencies.
```

Comment:
this shouldn't be a mere recommendation - it should be a step to be performed, that's why we strived so much to get all the deps from conda-forge, if a user starts mixing 50/50 PyPI/conda-forge deps they'll be in trouble in no time

Reply:
This section is intended for those who do not want to use conda at all

Reply:
ha! Good luck to them :rofl: Ok then